extensions-core/s3-extensions/src/main/java/org/apache/druid/storage/s3/S3TaskLogsConfig.java

```diff
@@ -35,13 +35,32 @@
   @NotNull
   private String s3Prefix = null;
 
+  @JsonProperty
+  private boolean disableAcl = false;
+
+  public void setDisableAcl(boolean disableAcl)
```

Comment:
This (and `setS3Bucket()`) don't strictly need to be public, but could be package-private (remove the modifier).

Since they're currently only being used by tests, annotating them with `@VisibleForTesting` would also be a good practice.

Reply:
Done.